.env-sample

```diff
@@ -2,4 +2,13 @@
 DEVNET_PRIVKEY=
 
 # Set L2 address here
```

Comment:
Do we need RPC config?  remove it if useless

Reply:
Now it is needed to deploy or test.